cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js

```diff
@@ -0,0 +1,64 @@
+import newOrder from '../../../support/fragments/orders/newOrder';
+import basicOrderLine from '../../../support/fragments/orders/basicOrderLine';
+import testType from '../../../support/dictionary/testTypes';
+import Orders from '../../../support/fragments/orders/orders';
+import Receiving from '../../../support/fragments/receiving/receiving';
+import TopMenu from '../../../support/fragments/topMenu';
+import Helper from '../../../support/fragments/finance/financeHelper';
+import InventorySearch from '../../../support/fragments/inventory/inventorySearch';
+import InteractorsTools from '../../../support/utils/interactorsTools';
+import OrdersHelper from '../../../support/fragments/orders/ordersHelper';
+import CheckInActions from '../../../support/fragments/check-in-actions/checkInActions';
+
+describe('orders: Receive piece from Order', () => {
+  const order = { ...newOrder.defaultOrder };
+  const orderLine = { ...basicOrderLine.defaultOrderLine };
+  let orderNumber;
+  const barcode = Helper.getRandomBarcode();
+  const caption = 'autotestCaption';
+
+  before(() => {
+    cy.getAdminToken();
+    cy.getOrganizationApi({ query: 'name="Amazon.com"' })
```

Comment:
Please move all values of query name parameter to variables. for example const organizationName = 'Amazon.com'

Reply:
done)